src/mintpy/cli/prep_aria.py

```diff
@@ -106,6 +107,20 @@ def create_parser(subparsers=None):
                       help='Name of the azimuth angle file.')
     geom.add_argument('-w','--water-mask', dest='waterMaskFile', type=str,
                       help='Name of the water mask file')
+
+    # correction layers: troposphereTotal, ionosphere, solidEarthTides
+    corr = parser.add_argument_group('corrections')
+    corr.add_argument('-ct', '--tropo', dest='tropoFile', type=str,
+                      help='Name of the Troposhere Delay stack file', default=None)
+    corr.add_argument('-ci', '--iono', dest='ionoFile', type=str,
+                    help='Name of the Ionosphere Delay stack file', default=None)
+    corr.add_argument('-cs', '--set', dest='setFile', type=str,
+                    help='Name of the Solid Earth Tides stack file', default=None)
+    corr.add_argument('--cluster', dest='cluster', type=str, choices={'local', 'pbs', None},
```

Comment:
do we need  arguments cluster and num-workers as network inversion of iono stack should be outside of this script